### PR TITLE
Add note related to Data URLs

### PR DESCRIPTION
**Pull request by chrisdavidmills**
Note added for Fx support as per https://bugzilla.mozilla.org/show_bug.cgi?id=1401895

### CODE REVIEW

File: http/data-url.json [hunk 1]
```diff
@@ -22,12 +22,24 @@
...
           },
-          "firefox": {
-            "version_added": true
-          },
-          "firefox_android": {
-            "version_added": true
-          },
+          "firefox": [
+            {
+              "version_added": "59",
```
Comment (teoli2003) [inline http/data-url.json]: Are you sure of this? It looks to me data URL worked long time ago, but their use as top-level URL restricted in 58.

File: http/data-url.json [hunk 2]
```diff
@@ -22,12 +22,24 @@
...
+              "notes": "Top-level navigation to <code>data://</code> URIs has been blocked (see <a href='https://www.fxsitecompat.com/en-CA/docs/2017/data-url-navigations-on-top-level-window-will-now-be-blocked/'>Data URL navigations on top level window will now be blocked</a> for more details)."
```
Comment (teoli2003) [inline http/data-url.json]: I would rather link to this: https://blog.mozilla.org/security/2017/11/27/blocking-top-level-navigations-data-urls-firefox-58/

File: http/data-url.json [hunk 3]
```diff
@@ -22,12 +22,24 @@
...
+            },
+            {
+              "version_added": true
+            }
+          ],
+          "firefox_android": [
+            {
+              "version_added": "59",
```
Comment (teoli2003) [inline http/data-url.json]: See above

File: http/data-url.json [hunk 4]
```diff
@@ -22,12 +22,24 @@
...
+              "notes": "Top-level navigation to <code>data://</code> URIs has been blocked (see <a href='https://www.fxsitecompat.com/en-CA/docs/2017/data-url-navigations-on-top-level-window-will-now-be-blocked/'>Data URL navigations on top level window will now be blocked</a> for more details)."
```
Comment (teoli2003) [inline http/data-url.json]: I would rather link to this: https://blog.mozilla.org/security/2017/11/27/blocking-top-level-navigations-data-urls-firefox-58/

**Comment by Elchi3** [comment]
I think this qualifies as a subfeature and not just a note. Other browsers might follow here and I think this would also deserve a proper section on the MDN page.

**Comment by chrisdavidmills** [comment]
OK, comments taken on board. I've put it in as a separate subfeature. And added a note on the MDN page about it.

**Comment by chrisdavidmills** [comment]
BTW, the relevant bug says Firefox 59 for this change. I think it was available before then but only in nightly/beta

**Pull request merged**